src/app/components/Button/Button.module.css

```diff
@@ -0,0 +1,28 @@
+.submit {
+  background-color: var(--main-font-color);
+  color: var(--secondary-font-color);
+  font: var(--main-font);
+  border: none;
+  border-radius: 0.25em;
```

Comment:
This is similar to the `.delete` style.

CSS modules supports compositions:
https://github.com/css-modules/css-modules#composition